lstchain/tools/lstchain_create_irf_files.py

```diff
@@ -484,11 +508,17 @@ def start(self):
                         'deg'
                     )
             else:
-                extra_headers["AL_CUT"] = (
-                    self.cuts.global_alpha_cut,
-                    'deg'
-                )
-
+                if self.energy_dependent_alpha:
+                    extra_headers["ALPHA_CONT"] = (
```

Comment:
I think there is still the 8 character limit on the HIERARCH card. In the latest astropy there is support for [longer keywords](https://docs.astropy.org/en/stable/io/fits/usage/headers.html#hierarch-cards), but we do not currently have the latest astropy.

Reply:
No, HIERARCH is specifically for longer than 8 character keywords. But it's an extension. We should try to fit in 8 chars.

Reply:
thanks, I forgot it... I changed it to `AL_CONT`